gui/include/scopy/gui/user_notes.hpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2019 Analog Devices Inc.
+ *
+ * This file is part of Scopy
+ * (see http://www.github.com/analogdevicesinc/scopy).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef USER_NOTES_HPP
+#define USER_NOTES_HPP
+
+#include <QButtonGroup>
+#include <QPushButton>
+#include <QSettings>
+#include <QTextBrowser>
+#include <QWidget>
+
+#include <scopy/core/api_object.hpp>
+#include <vector>
+
+using namespace std;
```

Comment:
This is considered bad practice bringing the whole std namespace, you could just use `std::vector` or instead of `using namespace std;` just `using std::vector;`